index.html

```diff
@@ -1,11 +1,90 @@
----
-layout: default
----
+<!DOCTYPE html>
```

Comment:
This looks like it's a duplicate of dist/index.html

Reply:
There's no longer an index.html in dist, dist only contains minified js and css.
Taking it out of dist solves the problem of hosting on github.